tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -555,7 +597,9 @@ async def test_recovering_view_after_restart_with_packet_loss(self, bft_network,
     @staticmethod
     async def _await_replicas_in_state_transfer(logger, bft_network, skvbc, primary):
         for r in bft_network.get_live_replicas():
+            logger.log_message("got live replicas")
             fetching = await bft_network.is_fetching(r)
+            logger.log_message(f"is_fetching: {fetching}")
```

Comment:
also here we can add the id of the replica:
`logger.log_message(f"{r} is_fetching: {fetching}")`